tests/python/relax/test_codegen_tir_cutlass.py

```diff
@@ -0,0 +1,754 @@
+                R.func_name("main")
+                A = R.arg(
+                    "A", relax.TensorStructInfo((batch, M, K), A_TYPE)
+                )  # pylint: disable=invalid-name
+                B = R.arg(
+                    "B", relax.TensorStructInfo((batch, K, N), B_TYPE)
+                )  # pylint: disable=invalid-name
+                bias = R.arg(
+                    "bias", relax.TensorStructInfo((1, N), A_TYPE)
+                )  # pylint: disable=invalid-name
+                with R.dataflow() as df:
+                    C = R.emit(R.matmul(A, B, out_dtype=C_TYPE))
+                    D = R.emit(R.add(C, bias))
+                    R.output(D)
+                (D,) = df.output_vars
+                R.func_ret_value(D)
+    relax_mod = ib.get()
+    return relax_mod
+
+
+@tvm.testing.requires_cutlass
+def test_cutlass_batch_dense2_bias():
+    b, m, n, k = 2, 128, 256, 64
+    executable = build(constructBatchGEMM2_bias(b, m, n, k))
+    dev = tvm.cuda()
+    A = np.random.randn(b, m, k).astype("float16")
+    B = np.random.randn(b, k, n).astype("float16")
+    bias = np.random.randn(1, n).astype("float16")
+    A_tvm = tvm.nd.array(A, dev)
+    B_tvm = tvm.nd.array(B, dev)
+    bias_tvm = tvm.nd.array(bias, dev)
+    result = f_run(executable, dev, A_tvm, B_tvm, bias_tvm)
+    np.testing.assert_allclose(result.numpy(), A @ B + bias, rtol=5e-2, atol=5e-2)
+
+
+def constructConv2D(N, C, H, W, KH, KW, O, strides, padding, dilation):
+    from tvm.script.ir_builder import IRBuilder
+    from tvm.script.ir_builder import ir as I
+    from tvm.script.ir_builder import relax as R
+    from tvm.script.ir_builder import tir as T
+
+    with IRBuilder() as ib:  # pylint: disable=invalid-name
+        with I.ir_module() as frame:
+            with R.function():
+                R.func_name("main")
+                x = R.arg(
+                    "x", relax.TensorStructInfo((N, H, W, C), A_TYPE)
+                )  # pylint: disable=invalid-name
+                w = R.arg(
+                    "w", relax.TensorStructInfo((O, KH, KW, C), B_TYPE)
+                )  # pylint: disable=invalid-name
+                with R.dataflow() as df:
+                    C = R.emit(
+                        R.nn.conv2d(
+                            x,
+                            w,
+                            strides=strides,
+                            padding=padding,
+                            dilation=dilation,
+                            groups=1,
+                            data_layout="NHWC",
+                            kernel_layout="OHWI",
+                            out_layout="NHWC",
+                            out_dtype=C_TYPE,
+                        )
+                    )
+                    R.output(C)
+                (C,) = df.output_vars
+                R.func_ret_value(C)
+    mod = ib.get()
+    return mod
+
+
+@tvm.testing.requires_cutlass
+def test_cutlass_conv2d():
+    import torch
```

Comment:
we shouldn't depend on torch for testing

Reply:
I see.